src/pages/CheckoutStepTwo.page.js

```diff
@@ -0,0 +1,44 @@
+import { getItemsListData } from '../additionalFunctions';
+import { ShoppingCartPage } from './ShoppingCart.page';
+
+export class CheckoutStepTwoPage extends ShoppingCartPage {
+    url = '/checkout-step-two.html';
+
+    get checkoutItems() { return this.page.locator('.cart_item'); }
```

Comment:
Its better to use css id or special test-data attributes in selectors everywhere if we could! Please pay attention to this comment!